.prettierrc.yml

```diff
@@ -0,0 +1,8 @@
+printWidth: 120
+tabWidth: 2
+useTabs: false
+semi: true
+singleQuote: true
+trailingComma: es5
+bracketSpacing: true
+arrowParens: avoid
```

Comment:
I am in favour of having a newline in the end for all files. I know this is a personal opinion, but by default we have all `.ts` files containing EOF newlines, so I believe we could extend for all files then.

Reply:
Fixed, I have added a new line to all of the config files which lacked it.